src/fragments/lib/auth/android/signin_next_steps/10_signin.mdx

```diff
@@ -105,15 +113,6 @@ try {
                       // flow, and once reset password flow completes, invoke
                       // `signIn` api to trigger signIn flow again.
                   }
```

Comment:
Should we also remove `RESET_PASSWORD` case?

Reply:
Yes, either here or in a separate PR.

Reply:
I am also going to update V1 docs.